x-pack/legacy/plugins/ml/public/data_frame/pages/job_management/components/job_list/transform_messages_pane.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment, useState, useEffect } from 'react';
+
+import { EuiSpacer, EuiBasicTable } from '@elastic/eui';
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
+import { i18n } from '@kbn/i18n';
+import { ml } from '../../../../../services/ml_api_service';
+
+const TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
+
+interface Props {
+  transformId: string;
+}
+
+export const TransformMessagesPane: React.SFC<Props> = ({ transformId }) => {
+  const [messages, setMessages] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  async function getMessages() {
+    try {
+      const messagesResp = await ml.dataFrame.getTransformAuditMessages(transformId);
+      setMessages(messagesResp);
+      setIsLoading(false);
+    } catch (error) {
+      setIsLoading(false);
+      setErrorMessage('Messages could not be loaded');
```

Comment:
do we translate error messages?

Reply:
Oh, yes - should do. Adding that in 👌 Thanks! 😄